internal/initwd/module_install_test.go

```diff
@@ -111,6 +111,67 @@ func TestModuleInstaller_error(t *testing.T) {
 	}
 }
 
+func TestModuleInstaller_symlink(t *testing.T) {
+	fixtureDir := filepath.Clean("test-fixtures/local-module-symlink")
+	dir, done := tempChdir(t, fixtureDir)
+	defer done()
+
+	hooks := &testInstallHooks{}
+
+	modulesDir := filepath.Join(dir, ".terraform/modules")
+	inst := NewModuleInstaller(modulesDir, nil)
+	_, diags := inst.InstallModules(".", false, hooks)
+	assertNoDiagnostics(t, diags)
+
+	wantCalls := []testInstallHookCall{
+		{
+			Name:        "Install",
+			ModuleAddr:  "child_a",
+			PackageAddr: "",
+			LocalPath:   "child_a",
+		},
+		{
+			Name:        "Install",
+			ModuleAddr:  "child_a.child_b",
+			PackageAddr: "",
+			LocalPath:   "child_a/child_b",
+		},
+	}
+
+	if assertResultDeepEqual(t, hooks.Calls, wantCalls) {
+		return
```

Comment:
What's happening here? If the calls and the desired calls are equal, we return, but then we don't get to the further asserts?

Reply:
No, the opposite. If they don't match it will return an error

Reply:
Oh, perfect, thanks, I didn't think to look at the implementation of the `assert`, agree on the intuitive-ness